lib/take_order.rb

```diff
@@ -0,0 +1,41 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness 
+require 'twilio-ruby'
+require './lib/order.rb'
+
+class Checkout
+
+  def initialize(money, total)
+    @money = money
+    @total = total#doesn't access order total, have to insert total
+    #upon initialising checkout
+  end
+
+def pay_money
+  if @money != @total
+    raise "You have entered an incorrect amount, please reenter"
+  else
+    puts "Thanks for your order! A text has been sent with your estimated delivery time."
+  end
+  send_message
```

Comment:
Avoid using puts. The brief does not ask for an interface. Does not really need an else.